picard/script/functions.py

```diff
@@ -1381,3 +1381,20 @@ def func_countryname(parser, country_code, translate=""):
     if translate:
         return gettext_countries(name)
     return name
+
+
+@script_function(eval_args=False, documentation=N_(
+    """`$is_multi(name)`
+
+Returns '1' if the argument is a multi-value tag and there are more than one elements, otherwise an empty string.
+
+Example:
+
+    $is_multi(%artists%)
+
+Result: 1 if there is more than one artist, otherwise "".
+"""
```

Comment:
When adding new function, I think we should always include Picard version it appeared.
As we did for example in https://github.com/metabrainz/picard/blob/89020eb8a14679ae2d677ea8d18b07873e9deb5d/picard/script/functions.py#L337